Add a basic rule to contributors / PR template. (#10277)
ray-project/ray: CONTRIBUTING.rst

PR Review Process
-----------------

For contributors who are in the ray-project organization:
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

- When you first create a PR, add an reviewer to the `assignee` section.
- Assignees will review your PR and add `@author-action-required` label if further actions are required.
- Address their comments and remove `@author-action-required` label from the PR.
- Repeat this process until assignees approve your PR.
- Once the PR is approved, the author is in charge of ensuring the PR passes the build. Add `test-ok` label if the build succeeds.

> - Committers will merge the PRs if the build success is confirmed.

Reviewer: ```suggestion
- Committers will merge the PR once the build is passing.
```

> For contributos who are out of the ray-project organization:

Reviewer: ```suggestion
For contributors who are not in the ray-project organization:
```
